src/installation/guides/arm-devices/index.md

```diff
@@ -149,10 +148,10 @@ better initial approximation of the current time, even without a network
 connection.
 
 **Warning**: Images from before 2019-11-09 might have an issue where the
```

Comment:
Perhaps we can modify this to use the date of the commit which fixed the issue?

Reply:
Sure. What's that date?

Reply:
Should be Mar 16: https://github.com/void-linux/void-mklive/pull/120

Reply:
Thanks, changed, and PR updated.